spec/rubocop/cop/rspec/file_fixture_spec.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::FileFixture, :config do
+  it 'registers an offense when using ' \
+     '`"\#{Rails.root}/spec/fixtures/files/some_file.csv"`' do
```

Comment:
This docstring just duplicates what the example itself tells. Does it make sense to write a free-form text explaining this case?